data_steward/analytics/cdr_ops/cross_ct_rt/output_prod_publish_qc.py

```diff
@@ -0,0 +1,446 @@
+                   dest_project_id=dest_project_id,
+                   src_dataset_id=src_dataset_id,
+                   dest_dataset_id=dest_dataset_id)
+execute(client, query)
+# -
+
+# ## Person vs Person_Ext in Destintation Dataset
+# Make sure the destination person and the destination person_ext tables have harmonious data for the five appended columns to the person table.
+# 1. sex_at_birth_concept_id
+# 2. sex_at_birth_source_concept_id,
+# 3. sex_at_birth_source_value
+# 4. state_of_residence_concept_id
+# 5. state_of_residence_source_value <br>
+#
+# Investigate any failed output.
+
+tpl = JINJA_ENV.from_string('''
+WITH calculation AS ( 
+SELECT 
+-- check state_of_residence columns --
+  COUNTIF(p.state_of_residence_concept_id <> pe.state_of_residence_concept_id) AS ne_state_of_residence_concept_id
+  ,COUNTIF((p.state_of_residence_concept_id IS NULL AND pe.state_of_residence_concept_id IS NOT NULL)
+      OR (p.state_of_residence_concept_id IS NOT NULL AND pe.state_of_residence_concept_id IS NULL)) as ne_nulls_state_of_residence_concept_id
+  ,COUNTIF(p.state_of_residence_source_value <> pe.state_of_residence_source_value) as ne_state_of_residence_source_value
+  ,COUNTIF((p.state_of_residence_source_value IS NULL AND pe.state_of_residence_source_value IS NOT NULL)
+      OR (p.state_of_residence_source_value IS NOT NULL AND pe.state_of_residence_source_value IS NULL)) as ne_nulls_state_of_residence_source_value
+-- check sex_at_birth columns --
+  ,COUNTIF(p.sex_at_birth_concept_id <> pe.sex_at_birth_concept_id) as ne_sex_at_birth_concept_id
+  ,COUNTIF((p.sex_at_birth_concept_id IS NULL AND pe.sex_at_birth_concept_id IS NOT NULL)
+      OR (p.sex_at_birth_concept_id IS NOT NULL AND pe.sex_at_birth_concept_id IS NULL)) as ne_nulls_sex_at_birth_concept_id
+  ,COUNTIF(p.sex_at_birth_source_concept_id <> pe.sex_at_birth_source_concept_id) as ne_sex_at_birth_source_concept_id
+  ,COUNTIF((p.sex_at_birth_source_concept_id IS NULL AND pe.sex_at_birth_source_concept_id IS NOT NULL)
+      OR (p.sex_at_birth_source_concept_id IS NOT NULL AND pe.sex_at_birth_source_concept_id IS NULL)) as ne_nulls_sex_at_birth_source_concept_id
+  ,COUNTIF(p.sex_at_birth_source_value <> pe.sex_at_birth_source_value) as ne_sex_at_birth_source_value
+  ,COUNTIF((p.sex_at_birth_source_value IS NULL AND pe.sex_at_birth_source_value IS NOT NULL)
+      OR (p.sex_at_birth_source_value IS NOT NULL AND pe.sex_at_birth_source_value IS NULL)) as ne_nulls_sex_at_birth_source_value
+FROM `{{dest_project_id}}.{{dest_dataset_id}}.person` p
+JOIN `{{dest_project_id}}.{{dest_dataset_id}}.person_ext` pe
+USING (person_id)
+)
+
+SELECT
+    'state_of_residence_concept_id_check' AS check
+    ,CASE
+        WHEN c.ne_state_of_residence_concept_id > 0
+          THEN 'FAILED'
+          ELSE 'passed'
+          END AS result
+FROM calculation AS c
+      
+UNION ALL 
+
+SELECT
+   'state_of_residence_concept_id_check' AS check
```

Comment:
I think this line should be:
`   'nulls_state_of_residence_concept_id_check' AS check`